.github/workflows/ddev-playwright.yml

```
@@ -79,6 +87,23 @@ jobs:
             JSON
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
+            
+      - name: Setup SSH keys
+        run: |
+            mkdir -p .ddev/homeadditions/.ssh
+            echo "${{ secrets.SSH_KEY }}" > .ddev/homeadditions/.ssh/id_rsa
+            echo "StrictHostKeyChecking=accept-new" > .ddev/homeadditions/.ssh/config
+            echo "CheckHostIP=no" > .ddev/homeadditions/.ssh/config
```

The `>` in line 96 overwrites the existing content of `.ddev/homeadditions/.ssh/config` written in line 95. You probably meant `>>` just like in the command on line 97.
Does it make sense to only use `>>`?
```suggestion
            echo "${{ secrets.SSH_KEY }}" >> .ddev/homeadditions/.ssh/id_rsa
            echo "StrictHostKeyChecking=accept-new" >> .ddev/homeadditions/.ssh/config
            echo "CheckHostIP=no" >> .ddev/homeadditions/.ssh/config
```

If it works as-is, does it mean that line 95 is unnecessary? 🤔

---

I think using `>` still makes sense because we know we created the file now.

Regarding `StrictHostKeyChecking=accept-new`, it is not required if we have "KNOWN_HOSTS" defined and only access those hosts from our container. Under different conditions, it is still useful because we can access remote destinations without `known_hosts` changes. DDEV has this setting by default.